webapp/common.js

```diff
@@ -41,7 +41,7 @@ function setWindow(window, back) {
           .removeAttr('href')
           .removeClass('button-hidden');
     } else {
-        backButton.attr('href', MESSAGES['issuers-overview-page']);
+        backButton.attr('href', 'javascript:history.length==1?window.close():history.back()');
         if (isInApp)
```

Comment:
The `isInApp` if-clause can be fully removed now. Also the declaration of the `isInApp` variable at the top of the javascript file can be removed.

Reply:
Although let me check how this looks like on not-updated devices.

Reply:
given old (not yet updated) apps, removing isInApp now is probably not a great idea, since window.close() probably wont work there

Reply:
In old versions this does not work, so let's keep it for now and remove it later. I'll make an issue for this.